sdks/python/apache_beam/io/fileio.py

```diff
@@ -270,6 +295,8 @@ def _rename_batch(batch):
       """_rename_batch executes batch rename operations."""
       source_files, destination_files = batch
       exceptions = []
+      if self._file_system is None:
```

Comment:
I am not sure what you mean, there is no file_path_prefix or file_name_suffix at that line (the enclosing fnc has the check).